File: Assignment__18/Question5.java
import java.util.Scanner;

class function
{
    char checkintegertype(int iNo)
    {
        if(iNo > 0)
        {
            return 'P';
        }
        else if(iNo < 0)
        {
           return 'N'; 
        }
        else
        {
            return 'Z';
        }
    }
}

class Question5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        char cResult = 0;

        function fobj = new function();
        
        cResult = fobj.checkintegertype(iValue);

        if(cResult == 'P')
        {
            System.out.println("The number is Positive.");
        }
        else if(cResult == 'N')
        {
            System.out.println("The number is Negative.");
        }
        else
        {
            System.out.println("The number is Zero.");
        }
    }
}

File: Assignment__40/Question5.c
#include<stdio.h>

void Display(int iNo, char ch)
{
    if(iNo == 0)   
    {
        return;
    }

    printf("%c\t", ch);     
    Display(iNo - 1, ch + 1); 
}

int main()
{
    int iValue = 0;
    char cValue = '\0';

    printf("Enter the number : ");
    scanf("%d", &iValue);

    printf("Enter the Character : ");
    scanf(" %c", &cValue);

    Display(iValue, cValue);

    return 0;
}
